docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+            if [ \$(head -1 results.txt | grep -c 'failed') -ne 0 ]; then exit 1; fi
+          """
+      }
+    }
+```
+
+### Github Actions
+
+**Create API Token Secret**
+
+Create a secret from the Repository Settings, for Sematext API Token.
+
+<img
+  class="content-modal-image"
+  alt="CI/CD GitHub Secret"
+  src="../../images/synthetics/ci-cd-github-secret.png"
+  title="Add Secret in Github"
+/>
+
+**Add Run Monitor Job to GitHub Action Workflow YAML**
+
+Create `.github/workflows/run-monitor.yml` GitHub Actions YAML to run your monitors on various CI/CD events. For example, to run the monitor after a deployment event, create a GitHub Action Job that gets executed on `deployment_status` event. The below action uses the `deployment_status.target_url` to pass the custom URL to the run monitor API.
+
+```yaml
+name: CI
+
+# Execute the run monitor job on deployment status event
+on:
+  deployment_status:
+
+jobs:
+  run_monitor:
+    runs-on: ubuntu-latest
+    outputs:
+      output1: ${{ steps.run.outputs.test }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Sematext Monitor
+        env:
+          API_TOKEN: ${{ secrets.SEMATEXT_API_TOKEN }}
+        id: run
+        run: |
+          target_url=`cat "$GITHUB_EVENT_PATH" | jq .deployment_status.target_url`
+          curl -s --request POST \
+            --url https://apps.sematext.com/synthetics-api/api/v3/apps/12345/monitors/runs \
+              --header 'authorization: apiKey '$API_TOKEN \
+                --header 'accept: text/plain' \
+                  --header 'content-type: application/json' \
+                    --data '[{"monitorId": 276, "url":'$target_url'}]' > results.txt
+          cat results.txt
+          if [ $(head -1 results.txt | grep -c 'failed') -ne 0 ]; then exit 1; fi
+```
+
+**GitHub Actions Logs**
+
+On every deployment event, the action will be invoked and the action logs will contain the results.
+
+<img
+  class="content-modal-image"
+  alt="CI/CD GitHub Secret"
+  src="../../images/synthetics/ci-cd-github-actions-log.png"
+  title="Add Secret in Github"
+/>
+
+
+### Bitbucket Pipelines
+
+**Create API Token Secret**
+
+Create a secret from the Repository Settings -> Repository Variables, for Sematext API Token.
+
+<img
+  class="content-modal-image"
+  alt="CI/CD BitBucket Secret"
+  src="../../images/synthetics/ci-cd-bitbucket-secret.png"
+  title="Add Secret in BitBucket"
+/>
+
+**Add Run Monitor Job to Bitbucket Pipeline Workflow YAML**
+
+Add Run Monitor step to `bitbucket-pipelines.yml`. Add the below steps after the deploy step in your pipeline configuration.
+
+```yaml
+image: node:10.15.3
+
+pipelines:
+  custom:
+    sematext:
+      - step:
+          script:
+            - curl -H "authorization:apiKey $SEMATEXT_API_TOKEN" -H "accept:text/plain" -H "content-type:application/json" -s -X POST -d "[{\"monitorId\":276}]" https://apps.sematext.com/synthetics-api/api/v3/apps/12345/monitors/runs > results.txt
```

Comment:
Are we referring to API *key* with a variable named SEMATEXT_API_TOKEN?

Reply:
Will change it to SEMATEXT_API_KEY and also update the screenshots